Websites/webkit.org/demos/webgpu/scripts/textured-cube-shader-constants.js

```diff
@@ -100,7 +100,7 @@ async function helloCube() {
         sampleCount: 1,
         dimension: "2d",
         format: "bgra8unorm",
-        usage: GPUTextureUsage.TEXTURE_BINDING | GPUTextureUsage.COPY_DST
+        usage: GPUTextureUsage.TEXTURE_BINDING | GPUTextureUsage.COPY_DST | GPUTextureUsage.RENDER_ATTACHMENT
```

Comment:
GPUTextureUsage.RENDER_ATTACHMENT shouldn't be needed, since texture or its views are not bound as render targets in this file as far as I could see